src/components/ShortcutGuide/shortcut.const.js

```diff
@@ -1,6 +1,19 @@
+const is_mac = OSnow();
+const control = is_mac ? "meta" : "ctrl";
+function OSnow() {
+  const agent = navigator.userAgent.toLowerCase();
+  const isMac = /macintosh|mac os x/i.test(navigator.userAgent);
+  if (agent.indexOf("win32") >= 0 || agent.indexOf("wow32") >= 0) {
+    return false;
+  } else if (agent.indexOf("win64") >= 0 || agent.indexOf("wow64") >= 0) {
+    return false;
+  } else if (isMac) {
+    return true;
+  }
+}
```

Comment:
I can't understand why the function is named as `0Snow`.  And you can know whether the OS is mac by `process.platform === "darwin"` 
```suggestion
const is_mac = process.platform === "darwin";
const control = is_mac ? "meta" : "ctrl";
```